### DISCUSSION TITLE
[A-L] (2023/24) Foglio 6 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#173] [Esercizi - Bertini]
![Screenshot 2023-11-06 173830](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/1fade32c-d5c2-46a1-9f45-2cb404906aed)

**Comment by Elia-Belli** [comment]
### Dado a 6 facce
Il lancio di un dado equo a 6 facce implica probabilità uniforme per tutti i 6 possibili risultati: $\mathbb{P}(X=x)=1/6,\forall x \in Im(X)$

<p align="center">
<img src=https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/da196a56-46f0-40ce-a15e-734c8d918a73 width=300>
</p>

$$\mathbb{E}(X)=\sum^6_{i=1}i\cdot 1/6=\frac{1}{6}\cdot \frac{6\cdot 7}{2}=3.5$$

$$\mathbb{V}(X)=\sum^6_{i=1}(i-3.5)^2\cdot 1/6=\frac{1}{6}\sum^6_{i=1}i^2-7i+3.5^2=\frac{1}{6}(91-7\cdot 21+6\cdot3.5^2)\simeq 2.92$$

---
### Dado ad n facce
Analogamente a prima $\mathbb{P}(X=x)=\frac{1}{n},\forall x \in Im(X)$
$$\mathbb{E}(X)=\sum^{n}_{i=1}i\cdot \frac{1}{n}=\frac{1}{n}\frac{n(n+1)}{2}=\frac{n+1}{2}$$

$$\mathbb{V}(X)=\sum^{n}\_{i=1}(i-\frac{n+1}{2})^2\cdot \frac{1}{n}=\frac{1}{n}\[\sum^n\_{i=1}i^2+\sum^n\_{i=1}(\frac{n+1}{2})^2-\sum^n_{i=1}2i\cdot \frac{n+1}{2}\]=$$

$$=\frac{1}{n}\[ \frac{n(n+1)(2n+1)}{6}+n(\frac{n+1}{2})^2-(n+1)\frac{n(n+1)}{2}\]=\frac{n^2-1}{12}$$

**Comment by CiottoloMaggico** [comment]
Analogo @Elia-Belli

**Comment by niccolo-fato** [comment]
![IMAGE 2024-02-17 15:33:37](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/73995334/55f5e185-daf5-4ed1-b7c4-9695556f8456)
